src/components/Discussion/Tree/CommentList.js

```diff
@@ -226,21 +264,28 @@ const CommentNode = ({ t, comment }) => {
    */
   const drawLineEnd = false
 
-  const rootStyle = styles.root({ isExpanded, nestLimitExceeded })
+  const rootStyle = styles.root({ isExpanded, nestLimitExceeded, depth, board })
+  const verticalToggleStyle = isRoot
+    ? null
```

Comment:
this is responsible for the regression with wired buttons above the name:
https://piratinnen.slack.com/archives/CSCC4NSJD/p1580830696006600
when `isRoot` both buttons where `verticalToggleStyle` is applied should not be rendered